pkg/compiler/compiler.go

```diff
@@ -262,3 +235,49 @@ func CompileAndSave(src string, o *Options) ([]byte, error) {
 
 	return b, nil
 }
+
+// CreateManifest creates manifest and checks that is is valid.
+func CreateManifest(di *DebugInfo, o *Options) (*manifest.Manifest, error) {
+	m, err := di.ConvertToManifest(o)
+	if err != nil {
+		return m, fmt.Errorf("failed to convert debug info to manifest: %w", err)
+	}
+	if !o.NoStandardCheck {
+		if err := standard.CheckABI(m, o.ContractSupportedStandards...); err != nil {
+			return m, err
+		}
+		if m.ABI.GetMethod(manifest.MethodOnNEP11Payment, -1) != nil {
```

Comment:
I think that we need to specify exactly the number of parameters which stndard's `OnNep11Payment` has (3). Because there might be a case when there are two `OnNep11Payment` methods with different params count in the contract.

Reply:
I think this is most likely a bug, which we can help to catch during compilation (like if someone forgets to add last `data` parameter).